lib/postgres/models/shop.go

```diff
@@ -0,0 +1,10 @@
+package models
+
+type Shop struct {
```

Comment:
GORM defaults to plural table name. To force table name to singular, u need to add a method to the model
```go
func (Shop) TableName() string {
	return "shop"
}
```